Thunder/bot/plugins/admin.py

```
@@ -258,9 +335,15 @@ async def run_shell_command(client: Client, message: Message):
         if len(output) > 4096:
             file = BytesIO(output.encode())
             file.name = "shell_output.txt"
-            await handle_flood_wait(message.reply_document, file, caption=MSG_SHELL_OUTPUT.format(command=html.escape(command)))
+            await handle_flood_wait(
+                message.reply_document, file,
+                caption=MSG_SHELL_OUTPUT.format(
+                    command=html.escape(command)))
         else:
             await reply(message, text=output, parse_mode=ParseMode.HTML)
             
     except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)